apps/framework-cli/src/cli/routines/metrics_console/run_console/app.rs

```diff
@@ -109,6 +175,25 @@ impl App {
                     .remove(0);
             }
         }
+        for path in &self.parsed_bytes_data.previous_bytes_in {
+            for item in path_bytes_in {
```

Comment:
You could have a `HashMap` instead of `Vec` here to simplify getting the rate and remove this loop